py3status/modules/watson.py

```diff
@@ -0,0 +1,87 @@
+"""
+Display the current status of the watson time-tracking tool.
+
+Configuration Parameters:
+    cache_time: Number of seconds before state is re-read (default: 10)
+    watson_state_file: Path to the file which watson uses to track its own state (default: '~/.config/watson/state')
+
+Requires:
+    https://github.com/TailorDev/Watson: commandline time tracking tool
+
+@author Markus Sommer (https://github.com/CryptoCopter)
+@license BSD
+
+SAMPLE OUTPUT
+{'full_text': 'Project baking [milk, eggs] started', 'color': '#00FF00'}
+"""
+
+import json
+import os
+
+from pathlib import Path
+from typing import Dict, List, Union
+
+
+class Py3status:
+    """
+    """
+
+    # available configuration parameters
+    cache_time = 10
+    watson_state_file = f"{Path.home()}/.config/watson/state"
```

Comment:
py3status still supports python3.5 so we cannot use fstrings yet

Reply:
also, tests are failing because the default written in the docstring `'~/.config/watson/state'` is not the same as here and it should be please

If you want to expand user home path, do it later on a `post_config_hook` method preferably

Reply:
Okay, then variable annotations need to go as well, these were only introduced in 3.6.
FYI: 3.5 reaches end-of-life on 2020-09-13 - so about a month from now.
Shall I make those changes and then submit another PR next month to undo them, or will you be supporting 3.5 past eol?

Reply:
also, what would be an acceptable default value for `state_file`?

Reply:
> Shall I make those changes and then submit another PR next month to undo them, or will you be supporting 3.5 past eol?

Yes please

Reply:
> also, what would be an acceptable default value for `state_file`?

check for instance the google_calendar module source code, the one in the doctstring sounds reasonable if that's the default watson is waiting for, else you could go for `~/.config/py3status/watson_state`

Reply:
okay, will do